src/OagBundle/Controller/ActivityController.php

```diff
@@ -44,7 +44,31 @@ public function enhanceAction(Request $request, OagFile $file, $iatiActivityId)
         $activity_detail['id'] = $srvActivity->getActivityId($activity);
         $activity_detail['name'] = $srvActivity->getActivityTitle($activity);
         $activity_detail['sectors'] = $srvActivity->getActivitySectors($activity);
-        $activity_detail['locations'] = $srvActivity->getActivityLocations($activity);
+
+        $locations = $srvActivity->getActivityLocations($activity);
+
+        $map_data = [];
```

Comment:
We're using a weird mixture of `array()` and `[]` in our services - we should decide on just one. Relates to @tobybatch 's suggestion that we stick to the Symfony coding standards